### DISCUSSION TITLE
请问存在分指令配置 Prefix 的操作吗

### DISCUSSION THREAD

**Discussion by HoiGe** [koishijs/koishi#1264] [Q&A] [answered]
不同的指令是否可以配置不同的 Prefix 呢

**Comment by MaikoTan** [comment, marked as answer]
No, but you could achieve that with [dialogue](https://dialogue.koishi.chat/zh-CN/) plugin instead:

Basic usage:
```
# /foo $(foo)
```

With default parameters:
```
# !bar $(bar -h)
```

Include all following text:
```
# "-baz (.*)$" $(baz $1)
```

**Reply by HoiGe** [reply]
Thank you

> Include all the following

Need to add brackets to work:

```# "-baz (.*)$" $(baz $1)```

**Reply by MaikoTan** [reply]
> > Include all the following
> 
> Need to add brackets to work:
> 
> ```# "-baz (.*)$" $(baz $1)```

Oh you're right, me updated the answer.
